src/index.html

```diff
@@ -16,7 +20,181 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo--sm">
+            <img src="images/icons/MyBiKE.svg" alt="Header Logo"
+             class="logo">
+          </a>
+          <a href="#" class="top-bar__logo--bg">
+            <img src="images/icons/MyBiKE-bg.svg" alt="Header Logo"
+             class="logo">
+          </a>
+          <div class="top-bar__icons">
+            <a href="#" class="icon icon--phone">‎</a>
+            <a href="#menu" class="icon icon--menu">‎</a>
+          </div>
+        </div>
+        <div class="header_bottom">
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="images/icons/MyBiKE-black.svg" alt="menu Logo"
+          class="logo">
+        </a>
+
+        <a href="#" class="icon icon--close">‎</a>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link">Home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#about" class="nav__link">About us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#compare-bikes" class="nav__link">Compare Bikes</a>
+            </li>
+            <li class="nav__item">
+              <a href="#details" class="nav__link">Details</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contact" class="nav__link">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel:+1 234 5555-55-55" class="nav__link-number">+1 234 5555-55-55</a>
+        <a href="tel:+1 234 5555-55-55" class="nav__link-book">Book a test ride</a>
+      </div>
+    </aside>
+
+    <main class="main ">
+      <div class="main__container">
+        <section class="move" id="about">
+          <h2 class="section-title section-title--align-left">Move Free</h2>
+          <p class="move__text">Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.</p>
+        </section>
+        <section class="compare-bikes" id="compare-bikes">
+          <h2 class="section-title">Compare Bikes</h2>
+          <div class="bikes__container">
+            <article class="bike">
+              <img src="images/bikes/bike-1.jpg" alt="Bike for sale"
```

Comment:
u need `a` tag as wrapper(`cursor: pointer` by default) + `scale` on hover, and if u want + shadows(FOR ALL IMAGES)
ur `a` tag should be with `#` in `href` attribute